pkg/kubernetes/scripts/date.js

```diff
@@ -24,13 +24,31 @@
 
     angular.module('kubernetes.date', [])
 
+    .directive('refreshEveryMin', [
+        '$interval',
+        function($interval) {
+            return {
+                restrict: 'E',
+                link: function(scope, element, attrs) {
+                    var interval = $interval(function () {}, 60000);
```

Comment:
I made the assumption that scheduling many more localized scope digests is cheaper than triggering a full rootScope digest that has potentially more expensive operations. But didn't profile, it's probably dependent on what the rest of the page is doing.

An alternative might be to have the loader trigger it's callbacks every min if it hasn't already, this already pretty much happens if you are watching nodes. WDYT?